lib/connection.js

```diff
@@ -193,6 +197,10 @@ class Connection extends EventEmitter {
   _heartbeatCallback(interval) {
     this.transport.send('{}');
     this.setTimeout(interval, this._heartbeatCallbackInstance);
+
+    if (process.env.NODE_ENV === 'debug') {
```

Comment:
I am not so sure about the "debug" value, as far as I know, "production" and "development" are used as values for the `NODE_ENV` environment variable.